files/zh-tw/web/http/reference/status/304/index.md

```diff
@@ -2,14 +2,23 @@
 title: 304 Not Modified
 slug: Web/HTTP/Reference/Status/304
 l10n:
-  sourceCommit: fb8c438501d119fb897a9667ee85c1486f2c03d6
+  sourceCommit: 4d929bb0a021c7130d5a71a4bf505bcb8070378d
 ---
 
 {{HTTPSidebar}}
 
-HTTP **`304 Not Modified`** 重新導向回應碼表示無需重新傳送所請求的資源。當請求是帶有 {{HTTPHeader("If-None-Match")}} 或 {{HTTPHeader("If-Modified-Since")}} 標頭的條件式 {{HTTPMethod("GET")}} 或 {{HTTPMethod("HEAD")}} 請求，並且該條件求值為 false 時，會發送此回應碼。如果條件求值為 true，則這是對一個已緩存的資源的隱式重定向，而該資源的請求如果沒有條件，則會導致一個 {{HTTPStatus("200")}} `OK` 的回應。
+HTTP **`304 Not Modified`** [重新導向回應](/zh-TW/docs/Web/HTTP/Reference/Status#重新導向訊息)狀態碼表示不需要重新傳送請求的資源。
 
-回應不得包含主體，並且必須包括與等效的 {{HTTPStatus("200")}} `OK` 回應中會發送的標頭：{{HTTPHeader("Cache-Control")}}、{{HTTPHeader("Content-Location")}}、{{HTTPHeader("Date")}}、{{HTTPHeader("ETag")}}、{{HTTPHeader("Expires")}} 和 {{HTTPHeader("Vary")}}。
+此狀態碼會在請求為[條件的](/zh-TW/docs/Web/HTTP/Guides/Conditional_requests) {{HTTPMethod("GET")}} 或 {{HTTPMethod("HEAD")}} 請求，且帶有 {{HTTPHeader("If-None-Match")}} 或 {{HTTPHeader("If-Modified-Since")}} 標頭，且條件判斷為「false」時傳送。這代表用戶端快取的資源仍然有效，如果條件為「true」則伺服器會回傳 {{HTTPStatus("200", "200 OK")}} 回應與資源。請參考 [HTTP 快取](/zh-TW/docs/Web/HTTP/Guides/Caching)以取得更多資訊。
```

Comment:
```suggestion
此狀態碼會在請求為帶有 {{HTTPHeader("If-None-Match")}} 或 {{HTTPHeader("If-Modified-Since")}} 標頭的 {{HTTPMethod("GET")}} 或 {{HTTPMethod("HEAD")}} [條件](/zh-TW/docs/Web/HTTP/Guides/Conditional_requests)請求，且條件判斷為「false」時傳送。這代表用戶端快取的資源仍然有效，如果條件為「true」則伺服器會回傳 {{HTTPStatus("200", "200 OK")}} 回應與資源。請參考 [HTTP 快取](/zh-TW/docs/Web/HTTP/Guides/Caching)以取得更多資訊。
```